Jenkinsfile

```diff
@@ -0,0 +1,70 @@
+@Library('jenkins.shared.library') _
+
+pipeline {
+  agent {
+    label 'ubuntu_docker_label'
+  }
+  environment {
+    HELM_IMAGE = "infoblox/helm:3.2.4-5b243a2"
+    VERSION = sh(script: "git describe --always --long --tags", returnStdout: true).trim()
+    TAG = "${env.VERSION}-j${env.BUILD_NUMBER}"
+  }
+  stages {
+    stage("Prepare Build") {
+      steps {
+        prepareBuild()
+      }
+    }
+    stage("Build Image") {
+      steps {
+        sh 'make build image VERSION=$TAG'
+      }
+    }
+    stage("Package Chart") {
+      steps {
+        withAWS(credentials: "CICD_HELM", region: "us-east-1") {
+          sh '''
+            docker run --rm \
```

Comment:
Do we need to rm this chart after it is pushed, or will it be automatically cleaned up?

Reply:
The Jenkins nodes are ephemeral, so I don't think cleanup is a big deal.